common/buildcraft/lib/net/MessageUpdateTile.java

```diff
@@ -46,6 +49,7 @@ public void fromBytes(ByteBuf buf) {
     public void toBytes(ByteBuf buf) {
         PacketBuffer buffer = new PacketBuffer(buf);
         buffer.writeBlockPos(pos);
+        buffer.writeString(className);
```

Comment:
There is a bug in network classes, so readPayload gives wrong buffer. It's for fix it for now and for debugging problem.